tests/forte/processors/data_augment/algorithms/embedding_similarity_replacement_op_test.py

```diff
@@ -93,24 +98,23 @@ def test_pipeline(self, texts, expected_outputs):
         nlp.add(component=WhiteSpaceTokenizer(), selector=AllPackSelector())
 
         processor_config = {
-            "augment_entry": "ft.onto.base_ontology.Token",
-            "other_entry_policy": {
-                "ft.onto.base_ontology.Document": "auto_align",
-                "ft.onto.base_ontology.Sentence": "auto_align",
-            },
-            "type": "data_augmentation_op",
             "data_aug_op": "forte.processors.data_augment.algorithms"
             ".embedding_similarity_replacement_op."
             "EmbeddingSimilarityReplacementOp",
             "data_aug_op_config": {
+                "other_entry_policy": {
+                    "ft.onto.base_ontology.Document": "auto_align",
+                    "ft.onto.base_ontology.Sentence": "auto_align",
+                },
+                "augment_entry": "ft.onto.base_ontology.Token",
                 "vocab_path": self.abs_vocab_path,
                 "embed_hparams": self.embed_hparams,
                 "top_k": 1,
             },
             "augment_pack_names": {"input": "augmented_input"},
         }
         nlp.add(
-            component=ReplacementDataAugmentProcessor(), config=processor_config
+            component=(BaseOpProcessor()), config=processor_config
```

Comment:
A side note to @Pushkar-Bhuse , `BaseOpProcessor` is not an intuitive name for people to use. They probably cannot guess this does data augmentation. A simple way is you provide an new name there by doing

```
DataAugProcssor = BaseOpProcssoor
```

in the implementation file

Reply:
I will look into this

Reply:
By the implementation file, are you referring to the test files where the `BaseOpProcessor` is used ?

Reply:
I feel that if we just want to change the name for the `BaseOpProcessor`. It will just be cleaner as we move forward with the new forward. I will make changes in all the places where it is used.